cluster/proxy-up.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Authors of Arktos - file modified.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Setup proxy for Arktos clusters
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+
+if [ -f "${KUBE_ROOT}/cluster/env.sh" ]; then
+    source "${KUBE_ROOT}/cluster/env.sh"
+fi
+
+source "${KUBE_ROOT}/cluster/kube-util.sh"
+
+if [ -z "${ZONE-}" ]; then
+  echo "... Creating proxy using provider: ${KUBERNETES_PROVIDER}" >&2
+else
+  echo "... Creating proxy in ${ZONE} using provider ${KUBERNETES_PROVIDER}" >&2
+fi
+
+echo "... calling verify-prereqs" >&2
+verify-prereqs
+echo "... calling verify-kube-binaries" >&2
+verify-kube-binaries
```

Comment:
does proxy use arktos-built binaries & tars?

Reply:
good point. only the prereq is needed here for the SSL and cert gen. the others are not needed as proxy cfg is built from source code on the fly.

Reply:
fixed.